src/data/polynomial/degree/basic.lean

```diff
@@ -733,6 +733,26 @@ theorem degree_le_iff_coeff_zero (f : polynomial R) (n : with_bot ℕ) :
 λ H, finset.sup_le $ λ b Hb, decidable.of_not_not $ λ Hn,
   (finsupp.mem_support_to_fun f b).1 Hb $ H b $ lt_of_not_ge Hn⟩
 
+theorem degree_lt_iff_coeff_zero (f : polynomial R) (n : ℕ) :
```

Comment:
Change looks good, please make it into a separate PR.

Reply:
#4792